doc/source/ray-contribute/development.rst

```diff
@@ -382,7 +382,7 @@ run the following (via ``-c`` ``fastbuild``/``dbg``/``opt``, respectively):
  bazel build -c fastbuild //:ray_pkg
 
 This will rebuild Ray with the appropriate options (which may take a while).
-If you need to build all targets, you can use ``"//:*"`` instead of
+If you need to build all targets, you can use ``"//:all"`` instead of
```

Comment:
I wonder if it actually means `//...` . `:all` means all targets under the root package.

Reply:
I don't thin `//...` will work out of the box though..

Reply:
`:*` includes files. `:all` does not.